File: 1884-minimum-changes-to-make-alternating-binary-string/minimum-changes-to-make-alternating-binary-string.cs
public class Solution {
    public int MinOperations(string s) {
        int n, c = 0, j = 0;
        for(n = 0; n < s.Length; n++, j ^= 1) {
            if(s[n] - '0' == j) c++;
        }
        return c < n - c ? c : n - c;
    }
}

File: 1884-minimum-changes-to-make-alternating-binary-string/minimum-changes-to-make-alternating-binary-string.java
class Solution {
    public int minOperations(String s) {
        int n, c = 0, j = 0;
        for(n = 0; n < s.length(); n++) {
            if(s.charAt(n) - '0' == j) c++;
            j ^= 1;
        }
        return c < n - c ? c : n - c;
    }
}